packages/field/src/FormRegistrarPortalMixin.js

```diff
@@ -0,0 +1,65 @@
+import { dedupeMixin } from '@lion/core';
+import { formRegistrarManager } from './formRegistrarManager.js';
+
+/**
+ * This will forward
```

Comment:
Please, explain a bit more what this Mixin does.

Reply:
added